### PR TITLE
improved spec for new_cases_spec

### PR DESCRIPTION
**Pull request by AspiringDev51**
Improving new_case_spec

### CODE REVIEW

File: spec/services/importers/new_cases_spec.rb [hunk 1]
```diff
@@ -2,8 +2,21 @@
...
+      file_path = 'spec/fixtures/importers/link.html'
+      html = File.read(file_path)
+      link_html = Nokogiri::HTML.parse(html)
+      link_html = link_html.css('a').first
+
+      data = described_class.new(link_html)
+      data.perform
+      expect(data.case_number).to eq 'CF-2021-489'
+      expect(data.case_types.keys.first).to eq 'CF'
+      expect(data.counties.keys.first).to eq 'Oklahoma'
```
Comment (holdenmitchell) [inline spec/services/importers/new_cases_spec.rb:19]: I think we want to directly query the database to make sure that the data has been saved. It should be saved, but if the `Importers::NewCourtCase.perform` method were to return the object without saving to the DB, this test would falsely pass

```ruby
court_case = CourtCase.find_by(county_id: county.id, oscn_id: oscn_id)

expect(court_case.case_number) to eq 'CF-2021-489'
# ...
```

File: spec/services/importers/new_cases_spec.rb [hunk 2]
```diff
@@ -2,8 +2,21 @@
...
     end
```
Comment (holdenmitchell) [inline spec/services/importers/new_cases_spec.rb:20]: I'm thinking there could be two different specs here:

1) Test that it creates the case when it does not exist in the database (What you currently have)
2) Test that it does not duplicate a case if it already exists in the database (Different spec)

**Review by holdenmitchell — CHANGES_REQUESTED** [review]
Left some small change requests! 🥈

File: spec/services/importers/new_cases_spec.rb [hunk 3]
```diff
@@ -2,8 +2,37 @@
...
+      data.perform
+      oscn_id = data.params['casemasterID']
+      court_case = CourtCase.find_by(county_id: county.id, oscn_id: oscn_id)
+
+      expect(court_case.case_number).to eq 'CF-2021-489'
+      expect(court_case.case_number[0..1]).to eq 'CF'
+      expect(court_case.county.name).to eq 'Oklahoma'
+    end
+
+    it ' does not duplicate' do
```
Comment (sgelbart) [inline spec/services/importers/new_cases_spec.rb:24]:
```suggestion
    it 'does not duplicate' do
```

**Review by sgelbart — APPROVED** [review]

File: app/services/importers/new_court_case.rb [hunk 4]
```diff
@@ -24,7 +24,7 @@ def perform
     private
 
     def oscn_id
-      params['casemasterid'].first.to_i
+      params['casemasterID'].first.to_i
```
Comment (sgelbart) [inline app/services/importers/new_court_case.rb:27]: did you mean to update this?

Comment (AspiringDev51) [reply app/services/importers/new_court_case.rb:27]: Yea, the param name was wrong, needed to be casemasterID

**Review by holdenmitchell — APPROVED** [review]

**Pull request merged**